src/templates/color-picker.hbs

```diff
@@ -27,11 +27,17 @@
     <form {{bind-attr class=":color-picker-custom-form isCustomColorValid:valid:invalid"}}>
       <div class="form-group form-group-sm">
         <div class="input-group">
-          <span {{bind-attr class=":input-group-addon
-            :color-picker-custom-preview isCustomColor:active"
-            style="customColorCSS"}}></span>
+          {{#if isCustomColorValid}}
```

Comment:
I'm guessing the suggestion is to put an action on the span and duplicate the click handler in `ColorPickerCell` in `ColorPicker` like 6709ee4?